api/internal/git/repospec.go

```diff
@@ -195,9 +198,31 @@ func peelQuery(arg string) (string, string, time.Duration, bool) {
 	return parsed.Path, ref, duration, submodules
 }
 
-func parseHostSpec(n string) (string, string) {
-	var host string
+func parseDomain(host string, schemes []string) string {
+	if host == "gh:" {
+		return "github.com"
+	}
+	domain := host
```

Comment:
For readability, let's name the variable `domain` to begin with. It's not clear why we pass it in as "host" and then suddenly change it to "domain".

Reply:
Did you mean rename the argument or move the variable assignment to the 1st line? I named the argument `host` to communicate that `parseDomain` expects the host (as `RepoSpec` names it) portion of a git url.

I changed the name to `target` and added comments. Open to other recs.